Dockerfile

```diff
@@ -8,12 +8,11 @@ RUN yum install -y \
 		make \
```

Comment:
Take a look at "Dockerfile.build" in this directory.  This is what we were using to make the build container for "build-containerized" before.

I like suffixing with ".build", because we also ship a `calico/ctl` container which should be the main Dockerfile in this repo (once we add it back in - don't have to do that in this PR!)